readthedocs/core/static-src/core/js/doc-embed/search.js

```diff
@@ -49,10 +50,9 @@ function attach_elastic_search_query(data) {
                             }
                         }
 
-                        // Creating the result from elements
-                        var link = doc.link + DOCUMENTATION_OPTIONS.FILE_SUFFIX + "?highlight=" + $.urlencode(query);
```

Comment:
I do worry about not using the local `DOCUMENTATION_OPTIONS.FILE_SUFFIX` -- are we sure that our API is returning the proper file path?

Reply:
I have undo this change. Earlier I got confused because of the `Http404: Directory indexes are not allowed here.` error because I am not serving the static files in local with nginx.